src/util.ts

```diff
@@ -7,3 +7,25 @@ export const randomNonce = (digits: number): string => {
   }
   return output;
 };
+
+export const randomString = (length = 6, adjust = 0): string => {
```

Comment:
Doesn't this do the same thing as `randomNonce` above? I am not sure what "adjust" does here though.